tests/integration-tests/tests/common/schedulers_common.py

```diff
@@ -288,7 +288,7 @@ def __init__(self, remote_command_executor):
             for value in ["EndTime=Unknown", "JobState=RUNNING", "JobState=COMPLETING", "JobState=CONFIGURING"]
         ),
         wait_fixed=seconds(3),
-        stop_max_delay=minutes(7),
+        stop_max_delay=minutes(14),
```

Comment:
a comment for how you got 14 minutes would be nice (e.g., twice the average that you saw when testing)

Reply:
Changed to 10 minutes. the longest test seems to be alltoall that takes about 9 minutes.